docs/resources/hvn_route.md

```diff
@@ -0,0 +1,139 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_hvn_route Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The HVN route resource allows you to manage an HVN route.
+---
+
+# hcp_hvn_route (Resource)
+
+The HVN route resource allows you to manage an HVN route.
+
+## Example Usage
```

Comment:
I think we should limit example to either TGW or peering, not both for simplicity. Peering would probably the best choice just because it is the most common type of connection and it's much simple.

Reply:
Yeah peering sounds good.

Reply:
done